blockchain/filedao/filedao_v2_util.go

```diff
@@ -172,21 +171,71 @@ func (fd *fileDAOv2) highestBlockOfStoreTip() uint64 {
 	return fd.header.Start + fd.blkStore.Size()*fd.header.BlockStoreSize - 1
 }
 
-func (fd *fileDAOv2) getBlockStore(height uint64) (*block.Store, error) {
+func (fd *fileDAOv2) getBlock(height uint64) (*block.Block, error) {
```

Comment:
adding block/receipt cache is good idea to improve read performance
can keep `getBlockStore()` to get even better performance
can you incorporate the changes under `/filedao` in https://github.com/iotexproject/iotex-core/commit/f1bc955c8eb58dba5f5b0e510f4b65770f8e1594 in this PR?

Reply:
check the top commit of `origin/v2dao` branch